themes/default/content/docs/intro/console/organizations.md

```diff
@@ -125,6 +125,29 @@ If you need help configuring or would like us to officially support another SAML
 provider,
 please [contact us]({{< relref "/about#contact-us" >}}).
 
+## Changing Membership Requirements
+
+Every organization is backed by an identity. The identity governs the membership to your org.
+When you add a new org to Pulumi, it uses the Pulumi identity by default. This means the membership is
+_only_ governed by a user having a Pulumi account and no additional identity requirements are placed on members.
+
+{{% notes %}}
+Regardless of the identity your org uses, org membership is always invite-only. Only the creator of the org gets automatic access. Everyone else must be invited by an admin to be admitted into the org.
+{{% /notes %}}
+
+However, if you want your org to mirror an org on a third party identity service such as GitHub, GitLab, or Bitbucket, you can
+change the backing identity. Enterprises can also choose SAML as the backing identity provider for an org as discussed in [SAML Single Sign-on](#saml-single-sign-on-sso). Changing your org's backing identity essentially changes the membership requirements you place
+on your members.
+
+Before you change your org's backing identity, ensure that all of its current members can satisfy the membership
+requirement of the org in the new identity service. That is, if you are switching from a GitHub-backed to a GitLab-backed
+org, ensure that all of your members are actually a member of the corresponding GitLab Group which your org would be
+changed to inherit from.
+
+To switch an org's backing identity you must be an org admin. Navigate to the org in the [Pulumi Console](https://app.pulumi.com)
+and go to the **Settings** page. Click on the **Access Management** navigation tab to the left and scroll down to the
+**Membership Requirements** card. Click on the button to get started with changing your org's identity.
```

Comment:
thanks for doing this work! a few important notes..

- i did a pass to remove "Click" from the console docs a bit ago as its pointer specific. we are using "select" now instead.
- i also believe we are trying to use "navigate" instead of "go to".
- it is also important that we avoid directional words, as folks that use screen readers do not have the same information about where things are placed on the page directionally.

```suggestion
To switch an organization's backing identity you must be an organization admin. Navigate to the organization's **Settings** page. Then navigate to **Access Management** and select the **Membership Requirements** button to get started with changing your organization's identity.

let me know if you have thoughts on a good place for this kind of documentation for future work.

```

Reply:
@susanev Thanks for submitting this suggestion! I was in the middle of writing up the same changes, but saw that you beat me to it. :)

Reply:
> let me know if you have thoughts on a good place for this kind of documentation for future work.

I can add it to the README of this repo with this PR.

Reply:
Done. Updated the PR